src/server.jsx

```diff
@@ -46,6 +47,22 @@ if (settings) {
 const supported = new locale.Locales(keys(languages), 'en');
 
 const server = express();
+// Internal proxy to bypass CORS while developing.
+// For the sake of simplicty, we only allow http://localhost:3000/api as the proxied api path
+if (__DEVELOPMENT__ && settings.apiPath.endsWith('http://localhost:3000/api')) {
```

Comment:
use settings.host, settings.port to check if the settings.apiPath is "local" to the Volto process